src/templates/converter.ts

```diff
@@ -57,23 +57,35 @@ export function convertToTsExpression({
 
 interface ConvertToFieldGroupInput {
     stringFields: FieldType[];
+    stringFieldsInSavedRecord: FieldType[];
     stringListFields: FieldType[];
     entityListFields: FieldType[];
     fileTypeFields: FieldType[];
 }
 function convertToFieldGroup(
-    input: ConvertToFieldGroupInput
+    input: ConvertToFieldGroupInput,
+    forSubtable: boolean = false
```

Comment:
@mura- Why is this flag needed?

Reply:
@pirosikick 
oh , Sorry, I thought that subtable fields don't have disabled property.
I will fix them. 🙏